init.sh

```diff
@@ -2,6 +2,15 @@
 #!/bin/sh -u
 # set FCCSWBASEDIR to the directory containing this script
 export FCCSWBASEDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# setup the pre-commit hook that checks with clang-format before committing
+# slc6 git version is too old for use with clang-format
+export PATH=/cvmfs/sft.cern.ch/lcg/contrib/git/bin:$PATH
+export LD_LIBRARY_PATH=/cvmfs/sft.cern.ch/lcg/contrib/git/lib64:$LD_LIBRARY_PATH
+if [ ! -L $FCCSWBASEDIR/.git/hooks/pre-commit ]; then
+  ln -s $FCCSWBASEDIR/FWCore/scripts/pre-commit.py $FCCSWBASEDIR/.git/hooks/pre-commit
```

Comment:
```suggestion
  mkdir -p $FCCSWBASEDIR/.git/hooks
  ln -s $FCCSWBASEDIR/git-hooks/pre-commit.py $FCCSWBASEDIR/.git/hooks/pre-commit
```

Reply:
@vvolkl The code above should work.

Also, I would not make `git-hooks` a hidden folder.